torchvision/transforms/v2/_augment.py

```diff
@@ -204,13 +204,20 @@ def _mixup_label(self, label: torch.Tensor, *, lam: float) -> torch.Tensor:
 
 
 class Mixup(_BaseMixupCutmix):
-    """[BETA] Apply Mixup to the provided batch of images and labels.
+    """[BETA] Apply MixUp to the provided batch of images and labels.
```

Comment:
Should we than also rename the classes?

Reply:
thought about it, went for consistency with timm (i.e. `Cutmix`). No strong opinion.

Reply:
I don't mind either, but I want both our documentation and class name to be the same. Your choice.

Reply:
:smiling_face_with_tear: https://github.com/pytorch/vision/pull/7731#discussion_r1277216830 I'm going to let you guys decide what to do as I want to avoid re-undo stuff.

(I don't think we absolutely have to align docs and code: CutMix is the technic, Cutmix is the class object - it's OK to have a distinction.)

Reply:
@vfdev-5 Could you clarify if you meant to only switch to `MixUp` / `CutMix` for the documentation or for the classes as well? I prefer to have both the same, but don't care whether it is `Cutmix` or `CutMix`.